code-reviews/recipes/Terraform.md

```diff
@@ -1,21 +1,33 @@
 # Terraform Code Reviews
 
-## Style guide
+## Style Guide
 
-[CSE](../../CSE.md) developers adhere to the [hashicorp style guide](https://www.terraform.io/docs/configuration/style.html).
+[CSE](../../CSE.md) developers follow the [hashicorp style guide.](https://www.terraform.io/docs/configuration/style.html)
```

Comment:
Nit: I'd keep the period out of the link - I guess it's matter of taste though.

Reply:
Updated